### PR TITLE
chore: cleanup codeowners

### PR DESCRIPTION
**Pull request by Amzani**
Fixes https://github.com/asyncapi/cli/issues/1287

### CODE REVIEW

**Review by Souvikns — DISMISSED** [review]
Hey @Amzani, welcome aboard! 🎉 🎉 🎉

**Comment by derberg** [comment]
@Amzani please have a look at https://github.com/asyncapi/cli/issues/668#issuecomment-1613183937

Can you demonstrate that you did some of this and therefore you understand the project you want to maintain?

**Comment by aeworxet** [comment]
@fmvilas trusted @Amzani to look over a Bounty Issue
https://github.com/asyncapi/studio/issues/755#issuecomment-1953820868
he translated design part of CLI into a technical task
https://github.com/asyncapi/cli/issues/1214
and he is an active maintainer of `Studio`
https://github.com/asyncapi/studio/pulls?q=is%3Apr+author%3AAmzani

**Comment by derberg** [comment]
@aeworxet thanks for clarification. Yup I'm aware Samir manages `Studio`. I also had a sync with @Souvikns that we need to onboard someone so I can quit as maintainers, and Samir's name popped up - which is completely fine. 

I just want to be fair and transparent and also consistent with message I share already: https://github.com/asyncapi/cli/issues/668#issuecomment-1613183937

I also proposed a good onboarding task that will be beneficial for the whole project, and great for maintainer onboarding -> https://github.com/asyncapi/cli/issues/781#issuecomment-1988082703

**Comment by Amzani** [comment]
@derberg makes sense!

(cc @smoya / @Barbanio - this might be also useful for you as you are designing the Maintainer mentorship program)

**Comment by derberg** [comment]
@Amzani I would close it, unless you are fine to update it to reflect https://github.com/asyncapi/cli/issues/1287 but of course once we have all voices there and agreement

**Comment by Amzani** [comment]
Sure @derberg I can update this one then and wait for @magicmatatjahu

**Comment by Amzani** [comment]
I left @magicmatatjahu for now until he is back from holidays, @derberg feel free to merge it or wait.

**Comment by aeworxet** [comment]
Can this PR be merged now?

File: CODEOWNERS [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
 
 # The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
-* @Souvikns @magicmatatjahu @derberg @asyncapi-bot-eve
+* @Souvikns @magicmatatjahu @Amzani @Shurtu-gal @asyncapi-bot-eve
```
Comment (derberg) [inline CODEOWNERS:8]:
```suggestion
* @Souvikns @Amzani @Shurtu-gal @asyncapi-bot-eve
```

Maciej also confirmed to be removed

**Review by derberg — APPROVED** [review]
@Amzani @Shurtu-gal you are now invited to repo

please do not forget to followup/triage together with @Souvikns what is already opened in the repo

@magicmatatjahu and me are removed from repo.

good luck!!!

**Pull request merged**